src/Sylius/Bundle/AddressingBundle/Form/Type/CountryType.php

```diff
@@ -28,12 +29,8 @@ class CountryType extends AbstractResourceType
     public function buildForm(FormBuilderInterface $builder, array $options)
     {
         $builder
-            ->add('translations', 'a2lix_translationsForms', array(
-                'form_type' => 'sylius_country_translation',
-                'label'    => 'sylius.form.country.name',
-            ))
-            ->add('isoName', 'text', array(
-                'label' => 'sylius.form.country.iso_name',
+            ->add('isoName', 'country', array(
+                'label' => 'sylius.form.country.name',
```

Comment:
It shows localized country name, not an iso name. It uses iso name as values in `option`.